app/tests/components/Review/Review.test.tsx

```diff
@@ -236,28 +236,6 @@ describe('The Project funding  section', () => {
     expect(
       document.getElementById('totalApplicantContribution')
     ).toHaveTextContent(`Total amount Applicant will contribute`);
-
-    expect(
-      document.getElementById('infrastructureBankFunding2223')
-    ).toHaveTextContent(`2022-23`);
-
-    expect(
-      document.getElementById('infrastructureBankFunding2324')
-    ).toHaveTextContent(`2023-24`);
-
-    expect(
-      document.getElementById('infrastructureBankFunding2425')
-    ).toHaveTextContent(`2024-25`);
-
-    expect(
-      document.getElementById('infrastructureBankFunding2526')
-    ).toHaveTextContent(`2025-26`);
-
-    expect(
-      document.getElementById('totalInfrastructureBankFunding')
-    ).toHaveTextContent(
-      `Total amount requested under Canadian Infrastructure Bank`
-    );
```

Comment:
We need a test to ensure that those now appear in the Other Funding Sources section

Reply:
Right, so we're waiting for the refactoring to be done to mark the issue complete then?

Reply:
Yeah I guess it's up to you if you want me to spend time updating this. In a hypothetical situation where the review refactor didn't get completed I suppose we would want this to be updated.